src/plugins/intel_cpu/src/nodes/gather.h

```diff
@@ -55,10 +58,14 @@ class Gather : public Node {
 private:
     void initShortParams(threadExecParams& p, uint64_t start);
     void execReference();
+    void fuseDecompressionConstant(const MemoryCPtr& memory, MemoryCPtr& decompressionValuesPtr);
 
     bool canOptimize1DCase = false;
     void exec1DCase();
 
+    bool canOptimizeWeightsDecompressionCase = false;
```

Comment:
change to `canOptimizeCompressedEmbedding` ?

Reply:
Done